node/src/main/scala/coop/rchain/node/node.scala

```diff
@@ -250,31 +250,31 @@ class NodeRuntime(conf: Configuration, host: String)(implicit scheduler: Schedul
         case handled       => handled.pure[Effect]
     }
 
-    for {
+    val info: Effect[Unit] = for {
       _ <- Log[Effect].info(
             s"RChain Node ${BuildInfo.version} (${BuildInfo.gitHeadCommit.getOrElse("commit # unknown")})")
+      _ <- if (conf.server.standalone) Log[Effect].info(s"Starting stand-alone node.")
+          else Log[Effect].info(s"Starting node that will bootstrap from ${conf.server.bootstrap}")
+    } yield ()
+
+    val loop: Effect[Unit] = for {
+      _ <- Connect.clearConnections[Effect]
```

Comment:
Clearing the connections every 5 seconds looks strange to me. I'm sure it makes sense based on what is actually happening, but maybe the naming could be better or there could be a comment explaining the reasoning?

Reply:
It's important how we clear it. When and how often is a matter of configuration and we can always update that. Most of the times `clearConnections` is a noop.